core/amber/src/main/resources/udf.conf

```diff
@@ -20,3 +20,8 @@ python {
         }
     }
 }
+
+r {
+    # Path to your R home here (if you want to use R-UDF)
+    path = ""
```

Comment:
If not specified, what is the default behavior? Does it try to find the R in the os PATH anyway?